presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -268,6 +268,27 @@ public void testTruncate()
         assertFunction("truncate(DECIMAL '1234567890123456789012.999')", createDecimalType(22, 0), SqlDecimal.of("1234567890123456789012"));
         assertFunction("truncate(DECIMAL '-1234567890123456789012.999')", createDecimalType(22, 0), SqlDecimal.of("-1234567890123456789012"));
 
+        // TRUNCATE DOUBLE -> DOUBLE
```

Comment:
I would put this into a new test case, let's call it `testTruncateWithPrecision`

Reply:
sure will do, thanks